### PR TITLE
added fbi command

### PR DESCRIPTION
**Pull request by meguminloli**

### CODE REVIEW

File: sadbot/app.py [hunk 1]
```diff
@@ -81,24 +81,25 @@ def send_message(
...
+                        continue
+                    messages.append({
                         "message": reply_message,
                         "parsemode": command["class"].parsemode,
-                    }
+                    }) 
             except re.error:
                 return None
-        return None
+        return messages
```
Comment (hydrastro) [inline sadbot/app.py:102]: Good idea!

**Review by hydrastro — COMMENTED** [review]
Looks good overall, there are some small things I need to do then we should be able to merge

File: sadbot/commands/channel.py [hunk 2]
```diff
@@ -31,19 +31,11 @@ def parsemode(self) -> Optional[str]:
...
-                req.text,
-            )
-            text = html.unescape(posts_list[0][1])
+            url = re.findall(r"https://boards.4channel.org/.*?/thread/[0-9]*", message.text)[0]
+            req = requests.get(url)
+            post = re.findall(r'post op".*?fileThu.*?img src=\"[/][/](.*?)\" alt.*?bloc.*?>(.*?)<[/]blo', req.text)[0]
+            text = html.unescape(post[1])
             text = markdownify.markdownify(text)
-            return text + "\n" + "https://" + posts_list[0][0] + "\n"
-        except (re.error, requests.ConnectionError):
```
Comment (hydrastro) [inline sadbot/commands/channel.py:48]: have you met an uncatched exception that crashed the bot?

Comment (meguminloli) [reply sadbot/commands/channel.py:48]: I never met one, but who knows...

**Comment by hydrastro** [comment]
Yoo

**Pull request merged**